panel/tests/ui/widgets/test_select.py

```diff
@@ -20,4 +20,4 @@ def test_select_option(page):
 
     serve_component(page, select)
 
-    assert page.locator('select').evaluate("(sel)=>sel.value")=='B'
+    wait_until(lambda: page.locator('select').evaluate("(sel)=>sel.value") == 'B', page)
```

Comment:
Hmmm. I thought "tricks" like these where not nescessary with Playwright @philippjfr ?

Reply:
If you use the expect API it isn't but I don't think that has a evaluate method.